src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts

```diff
@@ -56,7 +57,7 @@ export class ReplicationWizardComponent {
   saveSubmitText = T('START REPLICATION');
 
   protected entityWizard: any;
```

Comment:
seems like should be `EntityWizardComponent` type

Reply:
@denysbutenko Updated.